### ISSUE TITLE
Moolib Backend Issues

### ISSUE THREAD

**Issue by xiaomengy** [facebookresearch/rlmeta#65]
Recently there are several issues from moolib backend.
1. Based the observation of https://github.com/facebookresearch/moolib/issues/36, there is a performance regression in moolib.
2. There are several installation issues in moolib.

Based on this we are thinking about building another backend not using moolib. Open this issue to track the progress.

PR for gRPC backend: https://github.com/facebookresearch/rlmeta/pull/63

**Comment by xiaomengy** [comment]
After https://github.com/facebookresearch/moolib/pull/38 moolib fixed the performance issue by adding a new non TensorPipe backend. Based on this we can keep on using moolib and https://github.com/facebookresearch/rlmeta/pull/68 will fix that.

**Comment by xiaomengy** [comment]
The default backend of TorchRPC is also TensorPipe. The issue of TensorPipe is it has a large overhead especially when there are a large number of connections. So actually in the latest version of Moolib we have a new backend. But  I can try to compare them to provide some numbers here if you are interested.

**Comment by d3sm0** [comment]
Yes please, if it's not too much of an hustle. it would be interesting to see what is the trade-off of a custom rpc library, and how much of this performance gain come from the dynamic batching that one could implement on torch rpc. I also saw a branch with grpc.  What is the speedup from moolib compared to grpc?
